src/RaZ/Application.cpp

```diff
@@ -22,4 +26,14 @@ bool Application::run() {
   return m_isRunning && !m_activeWorlds.isEmpty();
 }
 
+void Application::run() {
```

Comment:
`run()` is defined before `runOnce()` in the header; it would be best to keep the same order